Realm/RLMObject.h

```diff
@@ -49,15 +51,17 @@ NS_ASSUME_NONNULL_BEGIN
  - `BOOL` or `bool`
  - `NSDate`
  - `NSData`
- - `NSNumber<X>`, where X is one of RLMInt, RLMFloat, RLMDouble or RLMBool, for optional number properties
+ - `NSNumber<X>`, where `X` is one of `RLMInt`, `RLMFloat`, `RLMDouble` or `RLMBool`, for optional number properties
  - `RLMObject` subclasses, so you can have many-to-one relationships.
- - `RLMArray<X>`, where X is an `RLMObject` subclass, so you can have many-to-many relationships.
+ - `RLMArray<X>`, where `X` is an `RLMObject` subclass, so you can have many-to-many relationships.
 
  ### Querying
  
- You can query an object directly via the class methods: `allObjects`, `objectsWhere:`, and `objectsWithPredicate:`.
- These methods allow you to easily query a custom subclass for instances of this class in the
- default Realm. To search in a Realm other than the default Realm use the interface on an RLMRealm instance.
+ You can initiate queries directly via the class methods: `allObjects`, `objectsWhere:`, and `objectsWithPredicate:`.
+ These methods allow you to easily query a custom subclass for instances of that class in the
+ default Realm. To search in a Realm other than the default Realm, use the interface on an `RLMRealm` instance.
```

Comment:
This last sentence has been wrong for a few years now (only the dynamic interface involves calling methods on RLMRealm instances for querying).